lib/glimesh/streams/channel.ex

```diff
@@ -44,6 +46,9 @@ defmodule Glimesh.Streams.Channel do
     field :poster, Glimesh.ChannelPoster.Type
     field :chat_bg, Glimesh.ChatBackground.Type
 
+    # This is used when searching for live channels that are live or hosted
```

Comment:
I use this virtual field on the "Following" page to determine if the matching stream was the result of the user directly following the live stream or the result of the user following an offline channel that is hosting a live stream.  Basically, this determines if I should put the "Hosted" gold badge on the stream's image on the page.